packages/icons/src/chaos-main.svg

```diff
@@ -0,0 +1,9 @@
+<svg viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M21.662 13.415C22.09 13.841 22.784 13.841 23.212 13.415C23.6401 12.989 23.6401 12.2983 23.212 11.8723L17.5653 6.25216L24.4024 2.32337C26.0013 1.40458 28 2.55307 28 4.39066L28 16.3974L19.7352 18.3653C19.1465 18.5055 18.7834 19.0941 18.9243 19.68C19.0651 20.266 19.6565 20.6273 20.2453 20.4872L28 18.6407V27.6093C28 29.4469 26.0013 30.5954 24.4024 29.6766L14.8901 24.2106L16.2212 14.6431C16.3042 14.0463 15.8854 13.4955 15.2858 13.4129C14.6862 13.3303 14.1328 13.7471 14.0498 14.3439L12.841 23.0331L4.1992 18.0673C2.60027 17.1485 2.60027 14.8515 4.1992 13.9327L15.5999 7.38152L21.662 13.415Z" fill="url(#paint0_linear_1522_106967)"/>
+<defs>
+<linearGradient id="paint0_linear_1522_106967" x1="12.7548" y1="6.36352" x2="27.8882" y2="29.9916" gradientUnits="userSpaceOnUse">
```

Comment:
gradients have a known issue with rendering incorrectly in Safari. Can we test it once?

Reply:
Ran the svg through SVGO and checked on safari, colors are rendering correctly.